libs/datasets/statistical_areas.py

```diff
@@ -0,0 +1,59 @@
+from dataclasses import dataclass
+from typing import Mapping
+import pandas as pd
+
+from libs import pipeline
+from libs.datasets import timeseries
+from libs.datasets.timeseries import MultiRegionTimeseriesDataset
+from covidactnow.datapublic.common_fields import CommonFields
+from libs.datasets import dataset_utils
+
+
+CBSA_LIST_PATH = "data/census-msa/list1_2020.xls"
+
+
+CBSA_COLUMN = "CBSA"
+
+
+@dataclass
+class CountyAggregator:
```

Comment:
May be helpful to mention aggregating to CBSA either in the class name or the `aggregate` function

Reply:
Done